mtgjson5/set_builder.py

```diff
@@ -1038,3 +1040,71 @@ def get_base_and_total_set_sizes(set_code: str) -> Tuple[int, int]:
     total_set_size = len(ScryfallProvider().download_cards(set_code))
 
     return base_set_size, total_set_size
+
+
+def add_tcgplayer_details(mtgjson_set: MtgjsonSetObject) -> MtgjsonSetObject:
+    """
+    Adds a list of all sku ids from TCGPlayer
+    :param mtgjson_set: MTGJSON Set
+    """
+
+    LOGGER.info(f"Adding TCGPlayer details for {mtgjson_set.code}")
+    list_of_product_ids: List
+    group_sku_data: Dict[str, List]
+    condition_map = {"1": "NM", "2": "LP", "3": "MP", "4": "HP", "5": "DMG", "6": "U"}
+    language_map = {
+        "1": "EN",
+        "2": "CS",
+        "3": "CT",
+        "4": "FR",
+        "5": "GE",
+        "6": "IT",
+        "7": "JP",
+        "8": "KR",
+        "9": "PT",
+        "10": "RU",
+        "11": "SP",
+    }
+    for mtgjson_card in mtgjson_set.cards:
+        if mtgjson_card.identifiers.tcgplayer_product_id is None:
+            continue
+        list_of_product_ids.append(mtgjson_card.identifiers.tcgplayer_product_id)
+    for counter in range(0, len(list_of_product_ids), 250):
+        section_list_product_id = list_of_product_ids[counter : counter + 250]
+        csv_product_ids = ",".join(section_list_product_id)
+        api_response = TCGPlayerProvider().download(
+            f"https://api.tcgplayer.com/[API_VERSION]/catalog/products/{csv_product_ids}?includeSkus=true"
+        )
+        for data in json.loads(api_response)["results"]:
+            translated_skus = []
+            product_id: str = ""
+            for sku in data["skus"]:
+                product_id = str(sku["productId"])
+                # Translates condition
+                condition = condition_map[str(sku["conditionId"])]
+                # Translated printing
+                if sku["printingId"] == 1:
+                    printing = "Normal"
+                else:
+                    printing = "Foil"
+                # Translates language
+                language = language_map[str(sku["languageId"])]
+                translated_skus.append(
+                    {
+                        "skuId": str(sku["skuId"]),
+                        "language": language,
+                        "printing": printing,
+                        "condition": condition,
+                    }
+                )
+            group_sku_data[product_id] = translated_skus
+            LOGGER.info(f"Adding TCGPlayer details for {translated_skus}")
+
+    for mtgjson_card in mtgjson_set.cards:
+        if mtgjson_card.identifiers.tcgplayer_product_id is None:
+            continue
+        mtgjson_card.identifiers.tcgplayer_sku_ids = group_sku_data[
+            mtgjson_card.identifiers.tcgplayer_product_id
+        ]
+
+    return mtgjson_set
```

Comment:
For this function, you don't actually have to return anything, as we are modifying an object in-place. You can see the return value isn't used by your change above where you simply call `add_tcgplayer_details(set) instead of set = add_tcgplayer_details(set)` :)